kabob-build/src/main/resources/edu/ucdenver/ccp/kabob/build/rules/bio_labels/genes.clj

```diff
@@ -0,0 +1,44 @@
+
+{:name "gene-labels-EntrezGene"
+
+ ;; The query pattern used to retrieve the data that will be used to construct
+ ;; the new triples.
+ :body
+ '(;; field value that contains the identifier
+   (_/gene_id_field kiao/hasTemplate iaoeg/EntrezGeneInfoFileData_geneIDDataField1)
+   (_/gene_id_field rdf/type kiao/FieldValue)
+   ;; the file record in which the field value appeared (record has-part
+   ;; field-value)
+   (_/record rdf/type iaoeg/EntrezGeneInfoFileData)
```

Comment:
Similar to the UniProt label rules, there is no bound variable on line 12 but there is on line 13, so swapping lines 12 and 13 may increase performance when using the `identity` clause reorderer, i.e when not permitting AG to reorder the clauses.